lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+require 'csv'
+require 'awesome_print'
+require 'pry'
+
+class Customer 
+
+  attr_reader :id
+  attr_accessor :email, :address
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+  
+  def self.all  
+  #CSV is an array of Hashes were each hash is a customer.
+  csv = CSV.read('/Users/dnsanche/ada/week3/grocery-store/data/customers.csv', headers: true).map(&:to_h) 
```

Comment:
you can't submit code that has the absolute path to github, because I don't have a folder called 'dnsanche'! Use the relative path moving forward.